unit/catch.cpp

```diff
@@ -0,0 +1,2 @@
+#define CATCH_CONFIG_MAIN
+#include "catch/catch.hpp"
```

Comment:
If `vendor/catch` is given as the include directory, this should just be:
```suggestion
#include "catch.hpp"
```